src/core/model.js

```diff
@@ -405,12 +405,12 @@ export default class Model extends Object3D {
           this.stats.accumulatedFrameTime / this.stats.profileFrameCount;
 
         // Log stats
-        log.log(2, 'program.id: ', this.program.id);
-        log.log(2, `last frame time: ${this.stats.lastFrameTime}ms`);
-        log.log(2, `average frame time ${this.stats.averageFrameTime}ms`);
-        log.log(2, `accumulated frame time: ${this.stats.accumulatedFrameTime}ms`);
-        log.log(2, `profile frame count: ${this.stats.profileFrameCount}`);
-
+        log.log(2, `\
+GPU time ${this.program.id}: ${this.stats.lastFrameTime}ms \
+avg ${this.stats.averageFrameTime}ms \
+acc: ${this.stats.accumulatedFrameTime}ms \
```

Comment:
Nit: should we use full names `average` and `accumulated` to be clear?

Reply:
> Nit: should we use full names average and accumulated to be clear?

We could. I was trying to keep this to a short line in the log. But probably the longer words could fit.